lib/dialyzer/test/abstract_SUITE.erl

```diff
@@ -82,7 +82,15 @@ generated_case(Config) when is_list(Config) ->
 		    {clause,[{location,7},{generated,true}],[{var,7,'_'}],[],
 		     [{call,7,{remote,7,{var,7,'Arg'},{atom,7,fn}},[]}]}]}]}]}],
 	     Config, [], []),
-    %% Location is a line (no column):
+   this_should_fail =  test([{attribute,{1,1},file,{"product_type.erl",1}},
```

Comment:
I had trouble compiling from main and running dialyzer tests, so I opted to just open this PR and get some guidance.

Reply:
I force pushed to address part of the problem here, but there's still an issue running tests locally, namely the init_per_suite fails.